content/en/blog/2023/announcing-major-release.md

```diff
@@ -0,0 +1,106 @@
+
+### Specifications
+
+[Notary Project specifications](https://github.com/notaryproject/specifications/releases/tag/v1.0.0) reach its first major release. All specifications, requirements, scenarios, threat model, and security audit reports are available in this release. ISVs and tool developers that want to interoperate with the Notary Project signatures and tooling should use the specifications to ensure compatibility.
+
+- [Notary Project OCI signature specification](https://github.com/notaryproject/specifications/blob/v1.0.0/specs/signature-specification.md)
+- [Notary Project OCI COSE signature envelope](https://github.com/notaryproject/specifications/blob/v1.0.0/specs/signature-envelope-cose.md)
+- [Notary Project OCI JWS signature envelope](https://github.com/notaryproject/specifications/blob/v1.0.0/specs/signature-envelope-jws.md)
+- [Notary Project OCI signing and verification workflow](https://github.com/notaryproject/specifications/blob/v1.0.0/specs/signing-and-verification-workflow.md)
+- [Notary Project signing scheme](https://github.com/notaryproject/specifications/blob/v1.0.0/specs/signing-scheme.md)
+- [Notary Project Trust Store and Trust Policy](https://github.com/notaryproject/specifications/blob/v1.0.0/specs/trust-store-trust-policy.md)
+- [Notation Plugin specification](https://github.com/notaryproject/specifications/blob/v1.0.0/specs/plugin-extensibility.md)
+
+### Signing and verification functionalities
+
+From the software producer's perspective, signing a software artifact is the way to ensure authenticity, anti-tampering, and increase trust when distributing software artifacts to consumers. Notary Project provides the following core capabilities for the signing experience.
+
+- Sign artifacts using signing keys stored securely in key management system (KMS). See the available plugins in the section [Extensibility: plugin support for Notation](#extensibility-plugin-support-for-notation)
+- Sign artifacts as well as list and inspect signatures stored in OCI-compliant registries
+    - Compliant with [`image-spec v1.0.2`](https://github.com/opencontainers/image-spec/tree/v1.0.2)
+    - Compliant with [`distribution-spec v1.0.1`](https://github.com/opencontainers/distribution-spec/tree/v1.0.1)
+    - Compatible with [`image-spec v1.1.0-rc4`](https://github.com/opencontainers/image-spec/tree/v1.1.0-rc4)
+    - Compatible with [`distribution-spec v1.1.0-rc3`](https://github.com/opencontainers/distribution-spec/tree/v1.1.0-rc3) (limited to [referrers tag schema](https://github.com/opencontainers/distribution-spec/blob/v1.1.0-rc3/spec.md#referrers-tag-schema))
+- Support two signature envelope formats 
+    - [COSE](https://github.com/notaryproject/notaryproject/blob/v1.0.0/specs/signature-envelope-cose.md): it provides message security assurance for services like IoT applications and using [CBOR](https://datatracker.ietf.org/doc/html/rfc8152) as the message-encoding format
```

Comment:
I'd drop the format descriptions and keep the links.

> - Supports the [COSE](https://github.com/notaryproject/notaryproject/blob/v1.0.0/specs/signature-envelope-cose.md) and [JWS](https://github.com/notaryproject/notaryproject/blob/v1.0.0/specs/signature-envelope-jws.md) signature envelope formats.

Reply:
I would keep this description and incorporate feedback from @toddysm since it helps users understand more about the signature envelopes and their benefits.